src/Materials/babylon.pbrMaterial.ts

```diff
@@ -1269,13 +1281,20 @@
 
                 // Colors
                 this._myScene.ambientColor.multiplyToRef(this.ambientColor, this._globalAmbientColor);
-                
-                // GAMMA CORRECTION.
-                this.convertColorToLinearSpaceToRef(this.reflectivityColor, PBRMaterial._scaledReflectivity);
+
+                if (this.metallic !== undefined || this.roughness !== undefined || this.metallicTexture) {
+                    var metallic = this.metallic === undefined ? 1 : this.metallic;

```

Comment:
Could you reuse the scaled reflectivity to avoid creating new color ?
 PBRMaterial._scaledReflectivity.r =